Parser/pegen.c

```diff
@@ -141,7 +141,8 @@ _create_dummy_identifier(Parser *p)
 static inline Py_ssize_t
 byte_offset_to_character_offset(PyObject *line, int col_offset)
 {
-    const char *str = PyUnicode_AsUTF8(line);
+    Py_ssize_t linesize;
+    const char *str = PyUnicode_AsUTF8AndSize(line, &linesize);
```

Comment:
Why does this make a difference, since `linesize` is not used anywhere thereafter?

Reply:
This was badly committed, give me a sec because I am still missing something

Reply:
Oh, sorry!